src/tests/calculator.test.js

```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react';
+import Calculator from '../components/calculator';
+
+describe('Should render as expected', () => {
+  test('test render of calculator page', () => {
+    const { container } = render(<Calculator />);
+    expect(container).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] @tjay1760 and @rbd3 Great work on your tests, however according to project requirements you are supposed to tests all components, kindly add tests for `App, InputButtons, OperationButtons, CalculatorPage, Output and QuotePage` components
![image](https://github.com/tjay1760/Math-Magician/assets/61408860/529187cb-c823-4198-8116-f7cca1d47104)